docs/tutorial.md

```diff
@@ -23,72 +23,86 @@ kernelspec:
 
 ## Toy example
 
-Suppose that we have observed the following data::
-
-    sample  alignment
-    0       A..G......C..G.....AT
-    1       T..G......A..C.....AG
-    2       A..G......A..C.....AC
-    3       A..C......C..G.....CT
-    4       A..C......C..G.....CT
-            |         |         |
-            0         10        20
-
-Here we have phased haplotype data for five samples at six variable sites. We wish to
-infer the genealogies that gave rise to this data set. To import the data into `tsinfer`
-we must know the *ancestral state* for each site we wish to use for inference; there are
-many methods for achieving this: details are outside the scope of this manual, but we
-have started a [discussion topic](https://github.com/tskit-dev/tsinfer/discussions/523)
-on this issue to provide some recommendations. Assuming that we
-know the ancestral states for most sites, we can then import our data into a `tsinfer`
-{ref}`Sample data<sec_file_formats_samples>` file using the Python
-{ref}`API<sec_api_file_formats>`:
+`Tsinfer` takes as input a [Zarr](https://zarr.readthedocs.io/) file, with phased variant data encoded in the
+[VCF Zarr](https://github.com/sgkit-dev/vcf-zarr-spec/) (.vcz) format. The standard
+route to create such a file is by conversion from a VCF file, e.g. using
+[vcf2zarr](https://sgkit-dev.github.io/bio2zarr/vcf2zarr/overview.html) as described later in this
+document. For a quick introduction, however, we will instead create an example file using
+[sgkit](https://sgkit-dev.github.io/sgkit/latest/).
+
 
 ```{code-cell} ipython3
+import sgkit
+ds = sgkit.simulate_genotype_call_dataset(n_variant=8, n_sample=3, missing_pct=0, phased=True, seed=79)
```

Comment:
I don't think we want to use sgkit apis, and certainly not simulate_genotype_call_dataset as it's fragile and not particularly useful. It would be better to have an example VCF and import it using vcf2zarr.

Reply:
OK, but we do that below. This bit of the tutorial was intended to introduce people to the tsinfer concept without having to deal with the ickiness of VCFs, I thought?

IMO it's better not to talk about specific formats like VCF, Plink, BGEN etc at the start, but keep it general? We could make our own zarr arrays, I suppose, but that's getting bogged down too quickly into detail.

Reply:
Also FWIW we currently round-trip the msprime data via VCF, but I'm hoping we can bin that once tskit2zarr is released, as it brings all sorts of extra dependencies (bgzip writer, tabix utility, etc).

Reply:
OK, In https://github.com/tskit-dev/tsinfer/pull/945/commits/319619b0e72ce5eac81f50d330f666fbb8221f23 I removed sgkit and simply load up an existing .vcz file at the start